new("DFrame", rownames = NULL, nrows = 39L, listData = list(query = c("rs1272061911", 
"rs6457401", "rs6500493", "rs6500493", "rs6500493", "rs4554318", 
"rs781627051", "rs781627051", "rs2240721", "rs12121609", "rs3135382", 
"rs3135382", "rs6420094", "rs587777600", "rs587777600", "rs12761149", 
"rs242561", "rs1053773776", "rs1053773776", "rs606231172", "rs183894680", 
"rs786203997", "rs4783186", "rs4783186", "rs121918106", "rs121918106", 
"rs218264", "rs218264", "rs2960321", "rs16891407", "rs16891407", 
"rs1005542482", "rs1005542482", "rs1052053", "rs1052053", "rs60820984", 
"rs60820984", "rs786204951", "rs11865131"), X_id = c("chr18:g.43460113C>T", 
"chr6:g.31334852G>T", "chr16:g.88803982C>T", "chr16:g.88803982C>A", 
"chr16:g.88803982C>G", "chr6:g.42012159C>T", "chr1:g.228353840A>G", 
"chr1:g.228353840A>C", "chr4:g.10020564G>A", "chr1:g.160070943C>T", 
"chr6:g.32383441T>A", "chr6:g.32383441T>G", "chr5:g.176817636A>G", 
"chr19:g.10426672T>G", "chr19:g.10426672T>A", "chr10:g.126605316G>A", 
"chr17:g.44026548T>C", "chr1:g.228353833A>T", "chr1:g.228353833A>G", 
"chr1:g.113499002_113499003insCCCCACCCCGCCACGTGACCGGCGT", "chr8:g.41655209C>T", 
"chr19:g.8386415A>G", "chr16:g.85415734T>A", "chr16:g.85415734T>C", 
"chr10:g.73610978T>C", "chr10:g.73610978T>A", "chr4:g.55408875A>T", 
"chr4:g.55408875A>G", "chr19:g.39048163C>A", "chr6:g.26206299T>A", 
"chr6:g.26206299T>C", "chr6:g.80816411A>G", "chr6:g.80816411A>T", 
"chr1:g.156202173A>G", "chr1:g.156202173A>C", "chr10:g.75639578C>T", 
"chr10:g.75639578C>G", "chr1:g.236917290A>G", "chr16:g.163667G>A"
), dbsnp._license = c("http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", "http://bit.ly/2AqoLOc", 
"http://bit.ly/2AqoLOc"), dbsnp.alleles = list(structure(list(
    allele = c("C", "T"), freq.gnomad_exomes = c(1, 0)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("G", "T"), freq.1000g = c(0.673, 
    0.327), freq.alspac = c(0.802, 0.198), freq.estonian = c(0.74, 
    0.26), freq.gnomad = c(0.706, 0.294), freq.northernsweden = c(0.783, 
    0.217), freq.page_study = c(0.625, 0.375), freq.topmed = c(0.709, 
    0.291), freq.twinsuk = c(0.789, 0.211)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("C", "G"), freq.1000g = c(0.243, 
    0.757), freq.alspac = c(0.32, 0.68), freq.estonian = c(0.343, 
    0.657), freq.exac = c(0.306, 0.694), freq.gnomad = c(0.295, 
    0.705), freq.northernsweden = c(0.295, 0.705), freq.topmed = c(0.283, 
    0.717), freq.twinsuk = c(0.32, 0.68), freq.vietnamese = c(0.123, 
    0.877)), class = "data.frame", row.names = 1:2), structure(list(
        allele = c("C", "G"), freq.1000g = c(0.243, 0.757), freq.alspac = c(0.32, 
        0.68), freq.estonian = c(0.343, 0.657), freq.exac = c(0.306, 
        0.694), freq.gnomad = c(0.295, 0.705), freq.northernsweden = c(0.295, 
        0.705), freq.topmed = c(0.283, 0.717), freq.twinsuk = c(0.32, 
        0.68), freq.vietnamese = c(0.123, 0.877)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("C", "G"), freq.1000g = c(0.243, 
    0.757), freq.alspac = c(0.32, 0.68), freq.estonian = c(0.343, 
    0.657), freq.exac = c(0.306, 0.694), freq.gnomad = c(0.295, 
    0.705), freq.northernsweden = c(0.295, 0.705), freq.topmed = c(0.283, 
    0.717), freq.twinsuk = c(0.32, 0.68), freq.vietnamese = c(0.123, 
    0.877)), class = "data.frame", row.names = 1:2), structure(list(
        allele = c("C", "T"), freq.1000g = c(0.531, 0.469), freq.alspac = c(0.534, 
        0.466), freq.estonian = c(0.469, 0.531), freq.gnomad = c(0.462, 
        0.538), freq.northernsweden = c(0.438, 0.562), freq.topmed = c(0.462, 
        0.538), freq.twinsuk = c(0.528, 0.472), freq.vietnamese = c(0.748, 
        0.252)), class = "data.frame", row.names = 1:2), structure(list(
        allele = c("A", "G"), freq.exac = c(1, 0), freq.gnomad_exomes = c(1, 
        0), freq.topmed = c(1, 0)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("A", "G"), freq.exac = c(1, 0), 
        freq.gnomad_exomes = c(1, 0), freq.topmed = c(1, 0)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("G", "A"), freq.1000g = c(0.591, 
    0.409), freq.alspac = c(0.447, 0.553), freq.estonian = c(0.475, 
    0.525), freq.exac = c(0.458, 0.542), freq.gnomad = c(0.56, 
    0.44), freq.gnomad_exomes = c(0.479, 0.521), freq.goesp = c(0.603, 
    0.397), freq.northernsweden = c(0.433, 0.567), freq.topmed = c(0.57, 
    0.43), freq.twinsuk = c(0.453, 0.547)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("C", "T"), freq.1000g = c(0.752, 
    0.248), freq.alspac = c(0.655, 0.345), freq.estonian = c(0.69, 
    0.31), freq.gnomad = c(0.711, 0.289), freq.northernsweden = c(0.655, 
    0.345), freq.topmed = c(0.728, 0.272), freq.twinsuk = c(0.656, 
    0.344), freq.vietnamese = c(0.778, 0.222)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("T", "G"), freq.1000g = c(0.929, 
    0.071), freq.alspac = c(0.82, 0.18), freq.estonian = c(0.823, 
    0.177), freq.gnomad = c(0.861, 0.139), freq.northernsweden = c(0.797, 
    0.203), freq.page_study = c(0.909, 0.091), freq.topmed = c(0.877, 
    0.123), freq.twinsuk = c(0.821, 0.179)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("T", "G"), freq.1000g = c(0.929, 
    0.071), freq.alspac = c(0.82, 0.18), freq.estonian = c(0.823, 
    0.177), freq.gnomad = c(0.861, 0.139), freq.northernsweden = c(0.797, 
    0.203), freq.page_study = c(0.909, 0.091), freq.topmed = c(0.877, 
    0.123), freq.twinsuk = c(0.821, 0.179)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("A", "G"), freq.1000g = c(0.744, 
    0.256), freq.alspac = c(0.667, 0.333), freq.estonian = c(0.659, 
    0.341), freq.gnomad = c(0.707, 0.293), freq.northernsweden = c(0.783, 
    0.217), freq.page_study = c(0.759, 0.241), freq.topmed = c(0.724, 
    0.276), freq.twinsuk = c(0.686, 0.314), freq.vietnamese = c(0.868, 
    0.132)), class = "data.frame", row.names = 1:2), NULL, NULL, 
    structure(list(allele = c("G", "A"), freq.1000g = c(0.439, 
    0.561), freq.alspac = c(0.255, 0.745), freq.estonian = c(0.257, 
    0.743), freq.gnomad = c(0.401, 0.599), freq.northernsweden = c(0.205, 
    0.795), freq.topmed = c(0.417, 0.583), freq.twinsuk = c(0.26, 
    0.74), freq.vietnamese = c(0.144, 0.856)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("T", "C"), freq.1000g = c(0.103, 
    0.897), freq.alspac = c(0.242, 0.758), freq.estonian = c(0.115, 
    0.885), freq.gnomad = c(0.141, 0.859), freq.northernsweden = c(0.118, 
    0.882), freq.page_study = c(0.114, 0.886), freq.topmed = c(0.167, 
    0.833), freq.twinsuk = c(0.234, 0.766)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("A", "G"), freq.gnomad = c(1, 0
    ), freq.topmed = c(1, 0)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("A", "G"), freq.gnomad = c(1, 0
    ), freq.topmed = c(1, 0)), class = "data.frame", row.names = 1:2), 
    NULL, structure(list(allele = c("C", "T"), freq.1000g = c(0.989, 
    0.011), freq.alspac = c(0.982, 0.018), freq.estonian = c(0.988, 
    0.012), freq.gnomad = c(0.985, 0.015), freq.northernsweden = c(0.982, 
    0.018), freq.topmed = c(0.984, 0.016), freq.twinsuk = c(0.979, 
    0.021)), class = "data.frame", row.names = 1:2), NULL, structure(list(
        allele = c("T", "C"), freq.1000g = c(0.198, 0.802), freq.alspac = c(0.128, 
        0.872), freq.estonian = c(0.142, 0.858), freq.gnomad = c(0.16, 
        0.84), freq.northernsweden = c(0.178, 0.822), freq.topmed = c(0.169, 
        0.831), freq.twinsuk = c(0.117, 0.883), freq.vietnamese = c(0.179, 
        0.821)), class = "data.frame", row.names = 1:2), structure(list(
        allele = c("T", "C"), freq.1000g = c(0.198, 0.802), freq.alspac = c(0.128, 
        0.872), freq.estonian = c(0.142, 0.858), freq.gnomad = c(0.16, 
        0.84), freq.northernsweden = c(0.178, 0.822), freq.topmed = c(0.169, 
        0.831), freq.twinsuk = c(0.117, 0.883), freq.vietnamese = c(0.179, 
        0.821)), class = "data.frame", row.names = 1:2), structure(list(
        allele = c("T", "C"), freq.exac = c(1, 0), freq.topmed = c(1, 
        0)), class = "data.frame", row.names = 1:2), structure(list(
        allele = c("T", "C"), freq.exac = c(1, 0), freq.topmed = c(1, 
        0)), class = "data.frame", row.names = 1:2), structure(list(
        allele = c("A", "T"), freq.1000g = c(0.572, 0.428), freq.alspac = c(0.738, 
        0.262), freq.estonian = c(0.771, 0.229), freq.gnomad = c(0.638, 
        0.362), freq.northernsweden = c(0.773, 0.227), freq.page_study = c(0.504, 
        0.496), freq.topmed = c(0.612, 0.388), freq.twinsuk = c(0.751, 
        0.249)), class = "data.frame", row.names = 1:2), structure(list(
        allele = c("A", "T"), freq.1000g = c(0.572, 0.428), freq.alspac = c(0.738, 
        0.262), freq.estonian = c(0.771, 0.229), freq.gnomad = c(0.638, 
        0.362), freq.northernsweden = c(0.773, 0.227), freq.page_study = c(0.504, 
        0.496), freq.topmed = c(0.612, 0.388), freq.twinsuk = c(0.751, 
        0.249)), class = "data.frame", row.names = 1:2), structure(list(
        allele = c("C", "A"), freq.1000g = c(0.745, 0.255), freq.alspac = c(0.889, 
        0.111), freq.estonian = c(0.914, 0.086), freq.gnomad = c(0.823, 
        0.177), freq.northernsweden = c(0.897, 0.103), freq.page_study = c(0.741, 
        0.259), freq.topmed = c(0.814, 0.186), freq.twinsuk = c(0.901, 
        0.099), freq.vietnamese = c(0.819, 0.181)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("T", "A"), freq.1000g = c(0.936, 
    0.064), freq.alspac = c(0.912, 0.088), freq.estonian = c(0.896, 
    0.104), freq.gnomad = c(0.93, 0.07), freq.northernsweden = c(0.897, 
    0.103), freq.topmed = c(0.938, 0.062), freq.twinsuk = c(0.92, 
    0.08), freq.vietnamese = c(0.976, 0.024)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("T", "A"), freq.1000g = c(0.936, 
    0.064), freq.alspac = c(0.912, 0.088), freq.estonian = c(0.896, 
    0.104), freq.gnomad = c(0.93, 0.07), freq.northernsweden = c(0.897, 
    0.103), freq.topmed = c(0.938, 0.062), freq.twinsuk = c(0.92, 
    0.08), freq.vietnamese = c(0.976, 0.024)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("A", "T"), freq.gnomad_exomes = c(1, 
    0), freq.topmed = c(1, 0)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("A", "T"), freq.gnomad_exomes = c(1, 
    0), freq.topmed = c(1, 0)), class = "data.frame", row.names = 1:2), 
    structure(list(allele = c("A", "C", "G"), freq.1000g = c(0.552, 
    0.001, 0.447), freq.alspac = c(0.62, NA, 0.38), freq.estonian = c(0.573, 
    NA, 0.427), freq.goesp = c(0.531, NA, 0.469), freq.northernsweden = c(0.622, 
    NA, 0.378), freq.page_study = c(0.478, NA, 0.522), freq.topmed = c(0.541, 
    0, 0.459), freq.twinsuk = c(0.607, NA, 0.393), freq.vietnamese = c(0.637, 
    NA, 0.363)), class = "data.frame", row.names = c(NA, 3L)), 
    structure(list(allele = c("A", "C", "G"), freq.1000g = c(0.552, 
    0.001, 0.447), freq.alspac = c(0.62, NA, 0.38), freq.estonian = c(0.573, 
    NA, 0.427), freq.goesp = c(0.531, NA, 0.469), freq.northernsweden = c(0.622, 
    NA, 0.378), freq.page_study = c(0.478, NA, 0.522), freq.topmed = c(0.541, 
    0, 0.459), freq.twinsuk = c(0.607, NA, 0.393), freq.vietnamese = c(0.637, 
    NA, 0.363)), class = "data.frame", row.names = c(NA, 3L)), 
    structure(list(allele = c("C", "T"), freq.1000g = c(0.814, 
    0.186), freq.alspac = c(0.821, 0.179), freq.estonian = c(0.838, 
    0.162), freq.gnomad = c(0.834, 0.166), freq.northernsweden = c(0.83, 
    0.17), freq.topmed = c(0.828, 0.172), freq.twinsuk = c(0.814, 
    0.186)), class = "data.frame", row.names = 1:2), structure(list(
        allele = c("C", "T"), freq.1000g = c(0.814, 0.186), freq.alspac = c(0.821, 
        0.179), freq.estonian = c(0.838, 0.162), freq.gnomad = c(0.834, 
        0.166), freq.northernsweden = c(0.83, 0.17), freq.topmed = c(0.828, 
        0.172), freq.twinsuk = c(0.814, 0.186)), class = "data.frame", row.names = 1:2), 
    NULL, structure(list(allele = c("G", "A"), freq.1000g = c(0.705, 
    0.295), freq.alspac = c(0.516, 0.484), freq.estonian = c(0.502, 
    0.498), freq.gnomad = c(0.616, 0.384), freq.northernsweden = c(0.532, 
    0.468), freq.page_study = c(0.721, 0.279), freq.topmed = c(0.633, 
    0.367), freq.twinsuk = c(0.494, 0.506), freq.vietnamese = c(0.799, 
    0.201)), class = "data.frame", row.names = 1:2)), dbsnp.alt = c("T", 
"T", "T", "T", "T", "T", "G", "G", "A", "T", "G", "G", "G", "G", 
"G", "A", "C", "T", "T", "CCCCACCCCGCCACGTGACCGGCGT", "T", "G", 
"C", "C", "C", "C", "T", "T", "A", "C", "C", "T", "T", "G", "G", 
"T", "T", "G", "A"), dbsnp.chrom = c("18", "6", "16", "16", "16", 
"6", "1", "1", "4", "1", "6", "6", "5", "19", "19", "10", "17", 
"1", "1", "1", "8", "19", "16", "16", "10", "10", "4", "4", "19", 
"6", "6", "6", "6", "1", "1", "10", "10", "1", "16"), dbsnp.dbsnp_build = c(153L, 
153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 
153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 
153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 153L, 
153L, 153L, 153L, 153L, 153L), dbsnp.gene = list(list(geneid = 57724L, 
    is_pseudo = FALSE, name = "ectopic P-granules autophagy protein 5 homolog", 
    rnas = structure(list(refseq = c("NM_020964.3", "XM_011526122.1", 
    "XM_017025889.1", "XM_017025890.2", "XM_017025891.1", "XM_017025892.1", 
    "XM_017025893.1", "XR_001753256.1", "XR_001753257.1"), so = list(
        structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
        structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
        structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
        structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
        structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
        structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
        structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
        structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L), 
        structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L)), 
        codon_aligned_transcript_change.deleted_sequence = c("TGC", 
        "TGC", "TGC", "TGC", "TGC", "TGC", "TGC", "G", "G"), 
        codon_aligned_transcript_change.inserted_sequence = c("TGC", 
        "TGC", "TGC", "TGC", "TGC", "TGC", "TGC", "G", "G"), 
        codon_aligned_transcript_change.position = c(5682L, 5692L, 
        5674L, 5674L, 5551L, 5717L, 3564L, 5675L, 5619L), codon_aligned_transcript_change.seq_id = c("NM_020964.3", 
        "XM_011526122.1", "XM_017025889.1", "XM_017025890.2", 
        "XM_017025891.1", "XM_017025892.1", "XM_017025893.1", 
        "XR_001753256.1", "XR_001753257.1"), protein.variant.spdi.deleted_sequence = c("C", 
        "C", "C", "C", "C", "C", "C", NA, NA), protein.variant.spdi.inserted_sequence = c("C", 
        "C", "C", "C", "C", "C", "C", NA, NA), protein.variant.spdi.position = c(1864L, 
        1864L, 1864L, 1864L, 1817L, 1509L, 739L, NA, NA), protein.variant.spdi.seq_id = c("NP_066015.2", 
        "XP_011524424.1", "XP_016881378.1", "XP_016881379.1", 
        "XP_016881380.1", "XP_016881381.1", "XP_016881382.1", 
        NA, NA), protein_product.refseq = c("NP_066015.2", "XP_011524424.1", 
        "XP_016881378.1", "XP_016881379.1", "XP_016881380.1", 
        "XP_016881381.1", "XP_016881382.1", NA, NA)), class = "data.frame", row.names = c(NA, 
    9L)), strand = "-", symbol = "EPG5"), NULL, structure(list(
    geneid = c(9780L, 100289580L), is_pseudo = c(FALSE, FALSE
    ), name = c("piezo type mechanosensitive ion channel component 1", 
    "uncharacterized LOC100289580"), rnas = list(structure(list(
        refseq = "NM_001142864.4", so = list(structure(list(accession = "SO:0001580", 
            name = "coding_sequence_variant"), class = "data.frame", row.names = 1L)), 
        codon_aligned_transcript_change.deleted_sequence = "GTC", 
        codon_aligned_transcript_change.inserted_sequence = "GTC", 
        codon_aligned_transcript_change.position = 1435L, codon_aligned_transcript_change.seq_id = "NM_001142864.4", 
        protein.variant.spdi.deleted_sequence = "V", protein.variant.spdi.inserted_sequence = "V", 
        protein.variant.spdi.position = 393L, protein.variant.spdi.seq_id = "NP_001136336.2", 
        protein_product.refseq = "NP_001136336.2"), class = "data.frame", row.names = 1L), 
        structure(list(refseq = "NR_103774.1", so = list(structure(list(
            accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1L)), 
    strand = c("-", "+"), symbol = c("PIEZO1", "LOC100289580"
    )), class = "data.frame", row.names = 1:2), structure(list(
    geneid = c(9780L, 100289580L), is_pseudo = c(FALSE, FALSE
    ), name = c("piezo type mechanosensitive ion channel component 1", 
    "uncharacterized LOC100289580"), rnas = list(structure(list(
        refseq = "NM_001142864.4", so = list(structure(list(accession = "SO:0001580", 
            name = "coding_sequence_variant"), class = "data.frame", row.names = 1L)), 
        codon_aligned_transcript_change.deleted_sequence = "GTC", 
        codon_aligned_transcript_change.inserted_sequence = "GTC", 
        codon_aligned_transcript_change.position = 1435L, codon_aligned_transcript_change.seq_id = "NM_001142864.4", 
        protein.variant.spdi.deleted_sequence = "V", protein.variant.spdi.inserted_sequence = "V", 
        protein.variant.spdi.position = 393L, protein.variant.spdi.seq_id = "NP_001136336.2", 
        protein_product.refseq = "NP_001136336.2"), class = "data.frame", row.names = 1L), 
        structure(list(refseq = "NR_103774.1", so = list(structure(list(
            accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1L)), 
    strand = c("-", "+"), symbol = c("PIEZO1", "LOC100289580"
    )), class = "data.frame", row.names = 1:2), structure(list(
    geneid = c(9780L, 100289580L), is_pseudo = c(FALSE, FALSE
    ), name = c("piezo type mechanosensitive ion channel component 1", 
    "uncharacterized LOC100289580"), rnas = list(structure(list(
        refseq = "NM_001142864.4", so = list(structure(list(accession = "SO:0001580", 
            name = "coding_sequence_variant"), class = "data.frame", row.names = 1L)), 
        codon_aligned_transcript_change.deleted_sequence = "GTC", 
        codon_aligned_transcript_change.inserted_sequence = "GTC", 
        codon_aligned_transcript_change.position = 1435L, codon_aligned_transcript_change.seq_id = "NM_001142864.4", 
        protein.variant.spdi.deleted_sequence = "V", protein.variant.spdi.inserted_sequence = "V", 
        protein.variant.spdi.position = 393L, protein.variant.spdi.seq_id = "NP_001136336.2", 
        protein_product.refseq = "NP_001136336.2"), class = "data.frame", row.names = 1L), 
        structure(list(refseq = "NR_103774.1", so = list(structure(list(
            accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1L)), 
    strand = c("-", "+"), symbol = c("PIEZO1", "LOC100289580"
    )), class = "data.frame", row.names = 1:2), list(geneid = 896L, 
    is_pseudo = FALSE, name = "cyclin D3", rnas = structure(list(
        refseq = c("NM_001136017.3", "NM_001136125.2", "NM_001136126.2", 
        "NM_001287427.1", "NM_001287434.1", "NM_001760.4", "XM_011514971.2", 
        "XM_011514972.1"), so = list(structure(list(accession = "SO:0001627", 
            name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L)), 
        protein_product.refseq = c("NP_001129489.1", "NP_001129597.1", 
        "NP_001129598.1", "NP_001274356.1", "NP_001274363.1", 
        "NP_001751.1", "XP_011513273.1", "XP_011513274.1")), class = "data.frame", row.names = c(NA, 
    8L)), strand = "-", symbol = "CCND3"), structure(list(geneid = c(200205L, 
574432L), is_pseudo = c(FALSE, FALSE), name = c("iron-sulfur cluster assembly factor IBA57", 
"IBA57 divergent transcript"), rnas = list(structure(list(refseq = c("NM_001010867.4", 
"NM_001310327.2"), so = list(structure(list(accession = "SO:0001580", 
    name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
    structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L)), 
    codon_aligned_transcript_change.deleted_sequence = c("TAT", 
    NA), codon_aligned_transcript_change.inserted_sequence = c("TAT", 
    NA), codon_aligned_transcript_change.position = c(334L, NA
    ), codon_aligned_transcript_change.seq_id = c("NM_001010867.4", 
    NA), protein.variant.spdi.deleted_sequence = c("Y", NA), 
    protein.variant.spdi.inserted_sequence = c("Y", NA), protein.variant.spdi.position = c(107L, 
    NA), protein.variant.spdi.seq_id = c("NP_001010867.1", NA
    ), protein_product.refseq = c("NP_001010867.1", "NP_001297256.1"
    )), class = "data.frame", row.names = 1:2), structure(list(
    refseq = c("NR_103539.1", "NR_103540.1"), so = list(structure(list(
        accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
        structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2)), 
    strand = c("+", "-"), symbol = c("IBA57", "IBA57-DT"), so = list(
        NULL, structure(list(accession = "SO:0001636", name = "2KB_upstream_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2), 
    structure(list(geneid = c(200205L, 574432L), is_pseudo = c(FALSE, 
    FALSE), name = c("iron-sulfur cluster assembly factor IBA57", 
    "IBA57 divergent transcript"), rnas = list(structure(list(
        refseq = c("NM_001010867.4", "NM_001310327.2"), so = list(
            structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L)), 
        codon_aligned_transcript_change.deleted_sequence = c("TAT", 
        NA), codon_aligned_transcript_change.inserted_sequence = c("TAT", 
        NA), codon_aligned_transcript_change.position = c(334L, 
        NA), codon_aligned_transcript_change.seq_id = c("NM_001010867.4", 
        NA), protein.variant.spdi.deleted_sequence = c("Y", NA
        ), protein.variant.spdi.inserted_sequence = c("Y", NA
        ), protein.variant.spdi.position = c(107L, NA), protein.variant.spdi.seq_id = c("NP_001010867.1", 
        NA), protein_product.refseq = c("NP_001010867.1", "NP_001297256.1"
        )), class = "data.frame", row.names = 1:2), structure(list(
        refseq = c("NR_103539.1", "NR_103540.1"), so = list(structure(list(
            accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2)), 
        strand = c("+", "-"), symbol = c("IBA57", "IBA57-DT"), 
        so = list(NULL, structure(list(accession = "SO:0001636", 
            name = "2KB_upstream_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2), 
    list(geneid = 56606L, is_pseudo = FALSE, name = "solute carrier family 2 member 9", 
        rnas = structure(list(refseq = c("NM_001001290.1", "NM_020041.3", 
        "XM_006713968.4", "XM_006713969.2", "XM_011513856.3", 
        "XM_011513857.1", "XM_011513858.1", "XM_011513859.3", 
        "XM_011513860.3", "XM_011513861.3", "XM_011513862.3", 
        "XM_011513864.2", "XM_011513865.2", "XM_011513866.2", 
        "XM_011513867.3", "XM_011513868.2", "XM_017008457.2", 
        "XM_017008458.2", "XM_017008459.1", "XM_017008460.2", 
        "XM_024454150.1", "XM_024454151.1", "XM_024454152.1", 
        "XM_024454153.1", "XR_001741290.1", "XR_001741291.1", 
        "XR_925341.3"), so = list(structure(list(accession = "SO:0001627", 
            name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L)), 
            protein_product.refseq = c("NP_001001290.1", "NP_064425.2", 
            "XP_006714031.1", "XP_006714032.1", "XP_011512158.1", 
            "XP_011512159.1", "XP_011512160.1", "XP_011512161.1", 
            "XP_011512162.1", "XP_011512163.1", "XP_011512164.1", 
            "XP_011512166.1", "XP_011512167.1", "XP_011512168.1", 
            "XP_011512169.1", "XP_011512170.1", "XP_016863946.1", 
            "XP_016863947.1", "XP_016863948.1", "XP_016863949.1", 
            "XP_024309918.1", "XP_024309919.1", "XP_024309920.1", 
            "XP_024309921.1", NA, NA, NA)), class = "data.frame", row.names = c(NA, 
        27L)), strand = "-", symbol = "SLC2A9"), list(geneid = 93185L, 
        is_pseudo = FALSE, name = "immunoglobulin superfamily member 8", 
        rnas = structure(list(refseq = c("NM_001206665.2", "NM_001320247.2", 
        "NM_052868.6", "XM_006711631.3", "XM_017002835.1", "XM_017002836.1", 
        "XM_017002837.1", "XM_017002838.1", "XM_017002839.1", 
        "XM_017002840.2", "XM_024450945.1", "XM_024450947.1"), 
            so = list(NULL, NULL, NULL, NULL, NULL, structure(list(
                accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                NULL, NULL, NULL, NULL, NULL, NULL), protein_product.refseq = c("NP_001193594.1", 
            "NP_001307176.1", "NP_443100.1", "XP_006711694.1", 
            "XP_016858324.1", "XP_016858325.1", "XP_016858326.1", 
            "XP_016858327.1", "XP_016858328.1", "XP_016858329.1", 
            "XP_024306713.1", "XP_024306715.1")), class = "data.frame", row.names = c(NA, 
        12L)), so = structure(list(accession = "SO:0001636", 
            name = "2KB_upstream_variant"), class = "data.frame", row.names = 1L), 
        strand = "-", symbol = "IGSF8"), NULL, NULL, list(geneid = 6569L, 
        is_pseudo = FALSE, name = "solute carrier family 34 member 1", 
        rnas = structure(list(refseq = c("NM_001167579.2", "NM_003052.5", 
        "XM_017009773.2", "XM_017009774.1", "XM_017009775.2", 
        "XM_024446191.1", "XR_941112.2", "XR_941113.2"), so = list(
            structure(list(accession = c("SO:0001987", "SO:0002152"
            ), name = c("downstream_transcript_variant", "genic_downstream_transcript_variant"
            )), class = "data.frame", row.names = 1:2), structure(list(
                accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002152", name = "genic_downstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L)), 
            protein_product.refseq = c("NP_001161051.1", "NP_003043.3", 
            "XP_016865262.1", "XP_016865263.1", "XP_016865264.1", 
            "XP_024301959.1", NA, NA), codon_aligned_transcript_change.deleted_sequence = c(NA, 
            NA, NA, NA, NA, NA, NA, "A"), codon_aligned_transcript_change.inserted_sequence = c(NA, 
            NA, NA, NA, NA, NA, NA, "A"), codon_aligned_transcript_change.position = c(NA, 
            NA, NA, NA, NA, NA, NA, 1931L), codon_aligned_transcript_change.seq_id = c(NA, 
            NA, NA, NA, NA, NA, NA, "XR_941113.2")), class = "data.frame", row.names = c(NA, 
        8L)), strand = "+", symbol = "SLC34A1"), structure(list(
        geneid = c(112812L, 125950L), is_pseudo = c(FALSE, FALSE
        ), name = c("ferredoxin 2", "ribonucleoprotein, PTB binding 1"
        ), rnas = list(structure(list(refseq = "NM_001031734.4", 
            so = list(structure(list(accession = "SO:0001580", 
                name = "coding_sequence_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = "ATG", 
            codon_aligned_transcript_change.inserted_sequence = "ATG", 
            codon_aligned_transcript_change.position = 19L, codon_aligned_transcript_change.seq_id = "NM_001031734.4", 
            protein.variant.spdi.deleted_sequence = "M", protein.variant.spdi.inserted_sequence = "M", 
            protein.variant.spdi.position = 3L, protein.variant.spdi.seq_id = "NP_001026904.2", 
            protein_product.refseq = "NP_001026904.2"), class = "data.frame", row.names = 1L), 
            structure(list(refseq = c("NM_001366174.1", "NM_133452.3"
            ), so = list(structure(list(accession = "SO:0001987", 
                name = "downstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001987", name = "downstream_transcript_variant"), class = "data.frame", row.names = 1L)), 
                protein_product.refseq = c("NP_001353103.1", 
                "NP_597709.3")), class = "data.frame", row.names = 1:2)), 
        strand = c("-", "-"), symbol = c("FDX2", "RAVER1"), so = list(
            NULL, structure(list(accession = "SO:0001634", name = "500B_downstream_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2), 
    structure(list(geneid = c(112812L, 125950L), is_pseudo = c(FALSE, 
    FALSE), name = c("ferredoxin 2", "ribonucleoprotein, PTB binding 1"
    ), rnas = list(structure(list(refseq = "NM_001031734.4", 
        so = list(structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L)), 
        codon_aligned_transcript_change.deleted_sequence = "ATG", 
        codon_aligned_transcript_change.inserted_sequence = "ATG", 
        codon_aligned_transcript_change.position = 19L, codon_aligned_transcript_change.seq_id = "NM_001031734.4", 
        protein.variant.spdi.deleted_sequence = "M", protein.variant.spdi.inserted_sequence = "M", 
        protein.variant.spdi.position = 3L, protein.variant.spdi.seq_id = "NP_001026904.2", 
        protein_product.refseq = "NP_001026904.2"), class = "data.frame", row.names = 1L), 
        structure(list(refseq = c("NM_001366174.1", "NM_133452.3"
        ), so = list(structure(list(accession = "SO:0001987", 
            name = "downstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001987", name = "downstream_transcript_variant"), class = "data.frame", row.names = 1L)), 
            protein_product.refseq = c("NP_001353103.1", "NP_597709.3"
            )), class = "data.frame", row.names = 1:2)), strand = c("-", 
    "-"), symbol = c("FDX2", "RAVER1"), so = list(NULL, structure(list(
        accession = "SO:0001634", name = "500B_downstream_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2), 
    NULL, list(geneid = 4137L, is_pseudo = FALSE, name = "microtubule associated protein tau", 
        rnas = structure(list(refseq = c("NM_001123066.3", "NM_001123067.3", 
        "NM_001203251.2", "NM_001203252.1", "NM_005910.5", "NM_016834.5", 
        "NM_016835.4", "NM_016841.5", "XM_005257362.4", "XM_005257364.4", 
        "XM_005257365.4", "XM_005257366.3", "XM_005257367.4", 
        "XM_005257368.4", "XM_005257369.4", "XM_005257370.4", 
        "XM_005257371.4"), so = list(structure(list(accession = "SO:0001627", 
            name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L)), 
            protein_product.refseq = c("NP_001116538.2", "NP_001116539.1", 
            "NP_001190180.1", "NP_001190181.1", "NP_005901.2", 
            "NP_058518.1", "NP_058519.3", "NP_058525.1", "XP_005257419.1", 
            "XP_005257421.1", "XP_005257422.1", "XP_005257423.1", 
            "XP_005257424.1", "XP_005257425.1", "XP_005257426.1", 
            "XP_005257427.1", "XP_005257428.1")), class = "data.frame", row.names = c(NA, 
        17L)), strand = "+", symbol = "MAPT"), structure(list(
        geneid = c(200205L, 574432L), is_pseudo = c(FALSE, FALSE
        ), name = c("iron-sulfur cluster assembly factor IBA57", 
        "IBA57 divergent transcript"), rnas = list(structure(list(
            refseq = c("NM_001010867.4", "NM_001310327.2"), so = list(
                structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = c("ACG", 
            NA), codon_aligned_transcript_change.inserted_sequence = c("ACG", 
            NA), codon_aligned_transcript_change.position = c(328L, 
            NA), codon_aligned_transcript_change.seq_id = c("NM_001010867.4", 
            NA), protein.variant.spdi.deleted_sequence = c("T", 
            NA), protein.variant.spdi.inserted_sequence = c("T", 
            NA), protein.variant.spdi.position = c(105L, NA), 
            protein.variant.spdi.seq_id = c("NP_001010867.1", 
            NA), protein_product.refseq = c("NP_001010867.1", 
            "NP_001297256.1")), class = "data.frame", row.names = 1:2), 
            structure(list(refseq = c("NR_103539.1", "NR_103540.1"
            ), so = list(structure(list(accession = "SO:0001986", 
                name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2)), 
        strand = c("+", "-"), symbol = c("IBA57", "IBA57-DT"), 
        so = list(NULL, structure(list(accession = "SO:0001636", 
            name = "2KB_upstream_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2), 
    structure(list(geneid = c(200205L, 574432L), is_pseudo = c(FALSE, 
    FALSE), name = c("iron-sulfur cluster assembly factor IBA57", 
    "IBA57 divergent transcript"), rnas = list(structure(list(
        refseq = c("NM_001010867.4", "NM_001310327.2"), so = list(
            structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L)), 
        codon_aligned_transcript_change.deleted_sequence = c("ACG", 
        NA), codon_aligned_transcript_change.inserted_sequence = c("ACG", 
        NA), codon_aligned_transcript_change.position = c(328L, 
        NA), codon_aligned_transcript_change.seq_id = c("NM_001010867.4", 
        NA), protein.variant.spdi.deleted_sequence = c("T", NA
        ), protein.variant.spdi.inserted_sequence = c("T", NA
        ), protein.variant.spdi.position = c(105L, NA), protein.variant.spdi.seq_id = c("NP_001010867.1", 
        NA), protein_product.refseq = c("NP_001010867.1", "NP_001297256.1"
        )), class = "data.frame", row.names = 1:2), structure(list(
        refseq = c("NR_103539.1", "NR_103540.1"), so = list(structure(list(
            accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2)), 
        strand = c("+", "-"), symbol = c("IBA57", "IBA57-DT"), 
        so = list(NULL, structure(list(accession = "SO:0001636", 
            name = "2KB_upstream_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2), 
    structure(list(geneid = c(6566L, 100506392L), is_pseudo = c(FALSE, 
    FALSE), name = c("solute carrier family 16 member 1", "SLC16A1 antisense RNA 1"
    ), rnas = list(structure(list(refseq = c("NM_001166496.1", 
    "NM_003051.3"), so = list(structure(list(accession = "SO:0001986", 
        name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
        structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L)), 
        protein_product.refseq = c("NP_001159968.1", "NP_003042.3"
        )), class = "data.frame", row.names = 1:2), structure(list(
        refseq = c("NR_103743.1", "NR_103744.1"), so = list(structure(list(
            accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2)), 
        so = list(structure(list(accession = "SO:0001636", name = "2KB_upstream_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001636", name = "2KB_upstream_variant"), class = "data.frame", row.names = 1L)), 
        strand = c("-", "+"), symbol = c("SLC16A1", "SLC16A1-AS1"
        )), class = "data.frame", row.names = 1:2), structure(list(
        geneid = c(286L, 105379392L), is_pseudo = c(FALSE, FALSE
        ), name = c("ankyrin 1", "uncharacterized LOC105379392"
        ), rnas = list(structure(list(refseq = c("NM_000037.4", 
        "NM_001142445.2", "NM_001142446.2", "NM_020475.3", "NM_020476.3", 
        "NM_020477.3", "NM_020478.5", "NM_020480.5", "XM_005273476.4", 
        "XM_011544490.3", "XM_011544491.3", "XM_011544494.3", 
        "XM_011544495.3", "XM_011544496.3", "XM_011544500.2", 
        "XM_011544501.2", "XM_011544502.2", "XM_011544503.3", 
        "XM_011544504.2", "XM_011544505.2", "XM_017013319.2", 
        "XM_017013320.2", "XM_017013321.1", "XM_017013322.1", 
        "XM_017013323.1", "XM_017013324.1", "XM_017013325.1", 
        "XM_017013326.1", "XM_017013327.2", "XM_017013328.2", 
        "XM_017013329.1", "XM_024447128.1"), so = list(structure(list(
            accession = c("SO:0001986", "SO:0002153"), name = c("upstream_transcript_variant", 
            "genic_upstream_transcript_variant")), class = "data.frame", row.names = 1:2), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = c("SO:0001986", "SO:0002153"
            ), name = c("upstream_transcript_variant", "genic_upstream_transcript_variant"
            )), class = "data.frame", row.names = 1:2), structure(list(
                accession = c("SO:0001986", "SO:0002153"), name = c("upstream_transcript_variant", 
                "genic_upstream_transcript_variant")), class = "data.frame", row.names = 1:2), 
            structure(list(accession = c("SO:0001986", "SO:0002153"
            ), name = c("upstream_transcript_variant", "genic_upstream_transcript_variant"
            )), class = "data.frame", row.names = 1:2), structure(list(
                accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = c("SO:0001986", "SO:0002153"
            ), name = c("upstream_transcript_variant", "genic_upstream_transcript_variant"
            )), class = "data.frame", row.names = 1:2), structure(list(
                accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = c("SO:0001986", "SO:0002153"
            ), name = c("upstream_transcript_variant", "genic_upstream_transcript_variant"
            )), class = "data.frame", row.names = 1:2)), protein_product.refseq = c("NP_000028.3", 
        "NP_001135917.1", "NP_001135918.1", "NP_065208.2", "NP_065209.2", 
        "NP_065210.2", "NP_065211.2", "NP_065213.2", "XP_005273533.1", 
        "XP_011542792.1", "XP_011542793.1", "XP_011542796.1", 
        "XP_011542797.1", "XP_011542798.1", "XP_011542802.1", 
        "XP_011542803.1", "XP_011542804.1", "XP_011542805.1", 
        "XP_011542806.1", "XP_011542807.1", "XP_016868808.1", 
        "XP_016868809.1", "XP_016868810.1", "XP_016868811.1", 
        "XP_016868812.1", "XP_016868813.1", "XP_016868814.1", 
        "XP_016868815.1", "XP_016868816.1", "XP_016868817.1", 
        "XP_016868818.1", "XP_024302896.1")), class = "data.frame", row.names = c(NA, 
        32L)), structure(list(refseq = c("XR_001745883.1", "XR_001745884.1", 
        "XR_949706.2", "XR_949707.2", "XR_949708.2"), so = list(
            structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = c(NA, 
        5L))), strand = c("-", "+"), symbol = c("ANK1", "LOC105379392"
        ), so = list(NULL, structure(list(accession = "SO:0001636", 
            name = "2KB_upstream_variant"), class = "data.frame", row.names = 1L))), class = "data.frame", row.names = 1:2), 
    structure(list(geneid = c(4701L, 6234L), is_pseudo = c(FALSE, 
    FALSE), name = c("NADH:ubiquinone oxidoreductase subunit A7", 
    "ribosomal protein S28"), rnas = list(structure(list(refseq = c("NM_005001.5", 
    "NR_135539.1"), so = list(structure(list(accession = "SO:0001986", 
        name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
        structure(list(accession = "SO:0001986", name = "upstream_transcript_variant"), class = "data.frame", row.names = 1L)), 
        protein_product.refseq = c("NP_004992.2", NA)), class = "data.frame", row.names = 1:2), 
        structure(list(refseq = "NM_001031.5", so = list(structure(list(
            accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = "ATG", 
            codon_aligned_transcript_change.inserted_sequence = "ATG", 
            codon_aligned_transcript_change.position = 35L, codon_aligned_transcript_change.seq_id = "NM_001031.5", 
            protein.variant.spdi.deleted_sequence = "M", protein.variant.spdi.inserted_sequence = "M", 
            protein.variant.spdi.position = 0L, protein.variant.spdi.seq_id = "NP_001022.1", 
            protein_product.refseq = "NP_001022.1"), class = "data.frame", row.names = 1L)), 
        so = list(structure(list(accession = "SO:0001636", name = "2KB_upstream_variant"), class = "data.frame", row.names = 1L), 
            NULL), strand = c("-", "+"), symbol = c("NDUFA7", 
        "RPS28")), class = "data.frame", row.names = 1:2), list(
        geneid = 23199L, is_pseudo = FALSE, name = "Gse1 coiled-coil protein", 
        rnas = structure(list(refseq = c("NM_001134473.3", "NM_001278184.3", 
        "NM_014615.5", "XM_005255859.5", "XM_005255860.3", "XM_005255861.5", 
        "XM_005255863.4", "XM_005255864.4", "XM_005255865.4", 
        "XM_011522965.3", "XM_011522966.3", "XM_017023083.1", 
        "XM_017023084.1", "XR_001751876.1", "XR_001751877.1"), 
            so = list(structure(list(accession = "SO:0002153", 
                name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002152", name = "genic_downstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002152", name = "genic_downstream_transcript_variant"), class = "data.frame", row.names = 1L)), 
            protein_product.refseq = c("NP_001127945.1", "NP_001265113.1", 
            "NP_055430.1", "XP_005255916.3", "XP_005255917.3", 
            "XP_005255918.3", "XP_005255920.1", "XP_005255921.1", 
            "XP_005255922.1", "XP_011521267.1", "XP_011521268.1", 
            "XP_016878572.1", "XP_016878573.1", NA, NA)), class = "data.frame", row.names = c(NA, 
        15L)), strand = "+", symbol = "GSE1"), list(geneid = 23199L, 
        is_pseudo = FALSE, name = "Gse1 coiled-coil protein", 
        rnas = structure(list(refseq = c("NM_001134473.3", "NM_001278184.3", 
        "NM_014615.5", "XM_005255859.5", "XM_005255860.3", "XM_005255861.5", 
        "XM_005255863.4", "XM_005255864.4", "XM_005255865.4", 
        "XM_011522965.3", "XM_011522966.3", "XM_017023083.1", 
        "XM_017023084.1", "XR_001751876.1", "XR_001751877.1"), 
            so = list(structure(list(accession = "SO:0002153", 
                name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002153", name = "genic_upstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002152", name = "genic_downstream_transcript_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0002152", name = "genic_downstream_transcript_variant"), class = "data.frame", row.names = 1L)), 
            protein_product.refseq = c("NP_001127945.1", "NP_001265113.1", 
            "NP_055430.1", "XP_005255916.3", "XP_005255917.3", 
            "XP_005255918.3", "XP_005255920.1", "XP_005255921.1", 
            "XP_005255922.1", "XP_011521267.1", "XP_011521268.1", 
            "XP_016878572.1", "XP_016878573.1", NA, NA)), class = "data.frame", row.names = c(NA, 
        15L)), strand = "+", symbol = "GSE1"), list(geneid = 5660L, 
        is_pseudo = FALSE, name = "prosaposin", rnas = structure(list(
            refseq = c("NM_001042465.3", "NM_001042466.3", "NM_002778.4"
            ), so = list(structure(list(accession = "SO:0001582", 
                name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = c("ATG", 
            "ATG", "ATG"), codon_aligned_transcript_change.inserted_sequence = c("ATG", 
            "ATG", "ATG"), codon_aligned_transcript_change.position = c(30L, 
            30L, 30L), codon_aligned_transcript_change.seq_id = c("NM_001042465.3", 
            "NM_001042466.3", "NM_002778.4"), protein.variant.spdi.deleted_sequence = c("M", 
            "M", "M"), protein.variant.spdi.inserted_sequence = c("M", 
            "M", "M"), protein.variant.spdi.position = c(0L, 
            0L, 0L), protein.variant.spdi.seq_id = c("NP_001035930.1", 
            "NP_001035931.1", "NP_002769.1"), protein_product.refseq = c("NP_001035930.1", 
            "NP_001035931.1", "NP_002769.1")), class = "data.frame", row.names = c(NA, 
        3L)), strand = "-", symbol = "PSAP"), list(geneid = 5660L, 
        is_pseudo = FALSE, name = "prosaposin", rnas = structure(list(
            refseq = c("NM_001042465.3", "NM_001042466.3", "NM_002778.4"
            ), so = list(structure(list(accession = "SO:0001582", 
                name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = c("ATG", 
            "ATG", "ATG"), codon_aligned_transcript_change.inserted_sequence = c("ATG", 
            "ATG", "ATG"), codon_aligned_transcript_change.position = c(30L, 
            30L, 30L), codon_aligned_transcript_change.seq_id = c("NM_001042465.3", 
            "NM_001042466.3", "NM_002778.4"), protein.variant.spdi.deleted_sequence = c("M", 
            "M", "M"), protein.variant.spdi.inserted_sequence = c("M", 
            "M", "M"), protein.variant.spdi.position = c(0L, 
            0L, 0L), protein.variant.spdi.seq_id = c("NP_001035930.1", 
            "NP_001035931.1", "NP_002769.1"), protein_product.refseq = c("NP_001035930.1", 
            "NP_001035931.1", "NP_002769.1")), class = "data.frame", row.names = c(NA, 
        3L)), strand = "-", symbol = "PSAP"), NULL, NULL, list(
        geneid = 6261L, is_pseudo = FALSE, name = "ryanodine receptor 1", 
        rnas = structure(list(refseq = c("NM_000540.2", "NM_001042723.2", 
        "XM_006723317.2", "XM_006723319.2", "XM_011527205.2", 
        "XR_001753735.1"), so = list(structure(list(accession = "SO:0001627", 
            name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0002152", name = "genic_downstream_transcript_variant"), class = "data.frame", row.names = 1L)), 
            protein_product.refseq = c("NP_000531.2", "NP_001036188.1", 
            "XP_006723380.1", "XP_006723382.1", "XP_011525507.1", 
            NA)), class = "data.frame", row.names = c(NA, 6L)), 
        strand = "+", symbol = "RYR1"), NULL, NULL, list(geneid = 594L, 
        is_pseudo = FALSE, name = "branched chain keto acid dehydrogenase E1 subunit beta", 
        rnas = structure(list(refseq = c("NM_000056.4", "NM_001318975.1", 
        "NM_183050.4", "NR_134945.1", "XM_005248756.5", "XM_011536023.3", 
        "XM_011536024.3", "XM_011536025.3", "XR_001743546.2", 
        "XR_001743547.2", "XR_001743548.2", "XR_001743549.2", 
        "XR_002956292.1"), so = list(structure(list(accession = "SO:0001582", 
            name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = c("ATG", 
            NA, "ATG", "A", "ATG", "ATG", "ATG", "ATG", "A", 
            "A", "A", "A", "A"), codon_aligned_transcript_change.inserted_sequence = c("ATG", 
            NA, "ATG", "A", "ATG", "ATG", "ATG", "ATG", "A", 
            "A", "A", "A", "A"), codon_aligned_transcript_change.position = c(84L, 
            NA, 23L, 84L, 30L, 30L, 30L, 30L, 30L, 30L, 30L, 
            30L, 30L), codon_aligned_transcript_change.seq_id = c("NM_000056.4", 
            NA, "NM_183050.4", "NR_134945.1", "XM_005248756.5", 
            "XM_011536023.3", "XM_011536024.3", "XM_011536025.3", 
            "XR_001743546.2", "XR_001743547.2", "XR_001743548.2", 
            "XR_001743549.2", "XR_002956292.1"), protein.variant.spdi.deleted_sequence = c("M", 
            NA, "M", NA, "M", "M", "M", "M", NA, NA, NA, NA, 
            NA), protein.variant.spdi.inserted_sequence = c("M", 
            NA, "M", NA, "M", "M", "M", "M", NA, NA, NA, NA, 
            NA), protein.variant.spdi.position = c(0L, NA, 0L, 
            NA, 0L, 0L, 0L, 0L, NA, NA, NA, NA, NA), protein.variant.spdi.seq_id = c("NP_000047.1", 
            NA, "NP_898871.1", NA, "XP_005248813.1", "XP_011534325.1", 
            "XP_011534326.1", "XP_011534327.1", NA, NA, NA, NA, 
            NA), protein_product.refseq = c("NP_000047.1", "NP_001305904.1", 
            "NP_898871.1", NA, "XP_005248813.1", "XP_011534325.1", 
            "XP_011534326.1", "XP_011534327.1", NA, NA, NA, NA, 
            NA)), class = "data.frame", row.names = c(NA, 13L
        )), strand = "+", symbol = "BCKDHB"), list(geneid = 594L, 
        is_pseudo = FALSE, name = "branched chain keto acid dehydrogenase E1 subunit beta", 
        rnas = structure(list(refseq = c("NM_000056.4", "NM_001318975.1", 
        "NM_183050.4", "NR_134945.1", "XM_005248756.5", "XM_011536023.3", 
        "XM_011536024.3", "XM_011536025.3", "XR_001743546.2", 
        "XR_001743547.2", "XR_001743548.2", "XR_001743549.2", 
        "XR_002956292.1"), so = list(structure(list(accession = "SO:0001582", 
            name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001582", name = "initiator_codon_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001619", name = "non_coding_transcript_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = c("ATG", 
            NA, "ATG", "A", "ATG", "ATG", "ATG", "ATG", "A", 
            "A", "A", "A", "A"), codon_aligned_transcript_change.inserted_sequence = c("ATG", 
            NA, "ATG", "A", "ATG", "ATG", "ATG", "ATG", "A", 
            "A", "A", "A", "A"), codon_aligned_transcript_change.position = c(84L, 
            NA, 23L, 84L, 30L, 30L, 30L, 30L, 30L, 30L, 30L, 
            30L, 30L), codon_aligned_transcript_change.seq_id = c("NM_000056.4", 
            NA, "NM_183050.4", "NR_134945.1", "XM_005248756.5", 
            "XM_011536023.3", "XM_011536024.3", "XM_011536025.3", 
            "XR_001743546.2", "XR_001743547.2", "XR_001743548.2", 
            "XR_001743549.2", "XR_002956292.1"), protein.variant.spdi.deleted_sequence = c("M", 
            NA, "M", NA, "M", "M", "M", "M", NA, NA, NA, NA, 
            NA), protein.variant.spdi.inserted_sequence = c("M", 
            NA, "M", NA, "M", "M", "M", "M", NA, NA, NA, NA, 
            NA), protein.variant.spdi.position = c(0L, NA, 0L, 
            NA, 0L, 0L, 0L, 0L, NA, NA, NA, NA, NA), protein.variant.spdi.seq_id = c("NP_000047.1", 
            NA, "NP_898871.1", NA, "XP_005248813.1", "XP_011534325.1", 
            "XP_011534326.1", "XP_011534327.1", NA, NA, NA, NA, 
            NA), protein_product.refseq = c("NP_000047.1", "NP_001305904.1", 
            "NP_898871.1", NA, "XP_005248813.1", "XP_011534325.1", 
            "XP_011534326.1", "XP_011534327.1", NA, NA, NA, NA, 
            NA)), class = "data.frame", row.names = c(NA, 13L
        )), strand = "+", symbol = "BCKDHB"), structure(list(
        geneid = c(11243L, 100527963L), is_pseudo = c(FALSE, 
        FALSE), name = c("polyamine modulated factor 1", "PMF1-BGLAP readthrough"
        ), rnas = list(structure(list(refseq = c("NM_001199653.1", 
        "NM_001199654.1", "NM_007221.3"), so = list(structure(list(
            accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = c("CAA", 
            NA, "CAA"), codon_aligned_transcript_change.inserted_sequence = c("CAA", 
            NA, "CAA"), codon_aligned_transcript_change.position = c(250L, 
            NA, 250L), codon_aligned_transcript_change.seq_id = c("NM_001199653.1", 
            NA, "NM_007221.3"), protein.variant.spdi.deleted_sequence = c("Q", 
            NA, "Q"), protein.variant.spdi.inserted_sequence = c("Q", 
            NA, "Q"), protein.variant.spdi.position = c(74L, 
            NA, 74L), protein.variant.spdi.seq_id = c("NP_001186582.1", 
            NA, "NP_009152.2"), protein_product.refseq = c("NP_001186582.1", 
            "NP_001186583.1", "NP_009152.2")), class = "data.frame", row.names = c(NA, 
        3L)), structure(list(refseq = c("NM_001199661.1", "NM_001199662.1", 
        "NM_001199663.1", "NM_001199664.1"), so = list(structure(list(
            accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = c("CAA", 
            "CAA", "CAA", NA), codon_aligned_transcript_change.inserted_sequence = c("CAA", 
            "CAA", "CAA", NA), codon_aligned_transcript_change.position = c(250L, 
            250L, 250L, NA), codon_aligned_transcript_change.seq_id = c("NM_001199661.1", 
            "NM_001199662.1", "NM_001199663.1", NA), protein.variant.spdi.deleted_sequence = c("Q", 
            "Q", "Q", NA), protein.variant.spdi.inserted_sequence = c("Q", 
            "Q", "Q", NA), protein.variant.spdi.position = c(74L, 
            74L, 74L, NA), protein.variant.spdi.seq_id = c("NP_001186590.1", 
            "NP_001186591.1", "NP_001186592.1", NA), protein_product.refseq = c("NP_001186590.1", 
            "NP_001186591.1", "NP_001186592.1", "NP_001186593.1"
            )), class = "data.frame", row.names = c(NA, 4L))), 
        strand = c("+", "+"), symbol = c("PMF1", "PMF1-BGLAP"
        )), class = "data.frame", row.names = 1:2), structure(list(
        geneid = c(11243L, 100527963L), is_pseudo = c(FALSE, 
        FALSE), name = c("polyamine modulated factor 1", "PMF1-BGLAP readthrough"
        ), rnas = list(structure(list(refseq = c("NM_001199653.1", 
        "NM_001199654.1", "NM_007221.3"), so = list(structure(list(
            accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = c("CAA", 
            NA, "CAA"), codon_aligned_transcript_change.inserted_sequence = c("CAA", 
            NA, "CAA"), codon_aligned_transcript_change.position = c(250L, 
            NA, 250L), codon_aligned_transcript_change.seq_id = c("NM_001199653.1", 
            NA, "NM_007221.3"), protein.variant.spdi.deleted_sequence = c("Q", 
            NA, "Q"), protein.variant.spdi.inserted_sequence = c("Q", 
            NA, "Q"), protein.variant.spdi.position = c(74L, 
            NA, 74L), protein.variant.spdi.seq_id = c("NP_001186582.1", 
            NA, "NP_009152.2"), protein_product.refseq = c("NP_001186582.1", 
            "NP_001186583.1", "NP_009152.2")), class = "data.frame", row.names = c(NA, 
        3L)), structure(list(refseq = c("NM_001199661.1", "NM_001199662.1", 
        "NM_001199663.1", "NM_001199664.1"), so = list(structure(list(
            accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = c("CAA", 
            "CAA", "CAA", NA), codon_aligned_transcript_change.inserted_sequence = c("CAA", 
            "CAA", "CAA", NA), codon_aligned_transcript_change.position = c(250L, 
            250L, 250L, NA), codon_aligned_transcript_change.seq_id = c("NM_001199661.1", 
            "NM_001199662.1", "NM_001199663.1", NA), protein.variant.spdi.deleted_sequence = c("Q", 
            "Q", "Q", NA), protein.variant.spdi.inserted_sequence = c("Q", 
            "Q", "Q", NA), protein.variant.spdi.position = c(74L, 
            74L, 74L, NA), protein.variant.spdi.seq_id = c("NP_001186590.1", 
            "NP_001186591.1", "NP_001186592.1", NA), protein_product.refseq = c("NP_001186590.1", 
            "NP_001186591.1", "NP_001186592.1", "NP_001186593.1"
            )), class = "data.frame", row.names = c(NA, 4L))), 
        strand = c("+", "+"), symbol = c("PMF1", "PMF1-BGLAP"
        )), class = "data.frame", row.names = 1:2), NULL, NULL, 
    list(geneid = 88L, is_pseudo = FALSE, name = "actinin alpha 2", 
        rnas = structure(list(refseq = c("NM_001103.3", "NM_001278343.2", 
        "NM_001278344.1"), so = list(structure(list(accession = "SO:0001580", 
            name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L), 
            structure(list(accession = "SO:0001580", name = "coding_sequence_variant"), class = "data.frame", row.names = 1L)), 
            codon_aligned_transcript_change.deleted_sequence = c("GAG", 
            "GAG", "GAG"), codon_aligned_transcript_change.inserted_sequence = c("GAG", 
            "GAG", "GAG"), codon_aligned_transcript_change.position = c(2101L, 
            2056L, 2298L), codon_aligned_transcript_change.seq_id = c("NM_001103.3", 
            "NM_001278343.2", "NM_001278344.1"), protein.variant.spdi.deleted_sequence = c("E", 
            "E", "E"), protein.variant.spdi.inserted_sequence = c("E", 
            "E", "E"), protein.variant.spdi.position = c(627L, 
            627L, 419L), protein.variant.spdi.seq_id = c("NP_001094.1", 
            "NP_001265272.1", "NP_001265273.1"), protein_product.refseq = c("NP_001094.1", 
            "NP_001265272.1", "NP_001265273.1")), class = "data.frame", row.names = c(NA, 
        3L)), strand = "+", symbol = "ACTN2"), list(geneid = 8131L, 
        is_pseudo = FALSE, name = "NPR3 like, GATOR1 complex subunit", 
        rnas = structure(list(refseq = c("NM_001039476.2", "NM_001077350.3", 
        "NM_001243247.1", "NM_001243248.1", "NM_001243249.1"), 
            so = list(structure(list(accession = "SO:0001627", 
                name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L), 
                structure(list(accession = "SO:0001627", name = "intron_variant"), class = "data.frame", row.names = 1L)), 
            protein_product.refseq = c("NP_001034565.1", "NP_001070818.1", 
            "NP_001230176.1", "NP_001230177.1", "NP_001230178.1"
            )), class = "data.frame", row.names = c(NA, 5L)), 
        strand = "-", symbol = "NPRL3")), dbsnp.ref = c("C", 
"G", "C", "C", "C", "C", "A", "A", "G", "C", "T", "T", "A", "T", 
"T", "G", "T", "A", "A", "", "C", "A", "T", "T", "T", "T", "A", 
"A", "C", "T", "T", "A", "A", "A", "A", "C", "C", "A", "G"), 
    dbsnp.rsid = c("rs1272061911", "rs6457401", "rs6500493", 
    "rs6500493", "rs6500493", "rs4554318", "rs781627051", "rs781627051", 
    "rs2240721", "rs12121609", "rs3135382", "rs3135382", "rs6420094", 
    "rs587777600", "rs587777600", "rs12761149", "rs242561", "rs1053773776", 
    "rs1053773776", "rs606231172", "rs183894680", "rs786203997", 
    "rs4783186", "rs4783186", "rs121918106", "rs121918106", "rs218264", 
    "rs218264", "rs2960321", "rs16891407", "rs16891407", "rs1005542482", 
    "rs1005542482", "rs1052053", "rs1052053", "rs60820984", "rs60820984", 
    "rs786204951", "rs11865131"), dbsnp.vartype = c("snv", "snv", 
    "snv", "snv", "snv", "snv", "snv", "snv", "snv", "snv", "snv", 
    "snv", "snv", "snv", "snv", "snv", "snv", "snv", "snv", "ins", 
    "snv", "snv", "snv", "snv", "snv", "snv", "snv", "snv", "snv", 
    "snv", "snv", "snv", "snv", "snv", "snv", "snv", "snv", "snv", 
    "snv"), dbsnp.dbsnp_merges = list(NULL, structure(list(date = c("2006-03-11T07:58Z", 
    "2011-09-17T04:07Z", "2012-05-4T12:46Z", "2010-08-16T13:01Z"
    ), rsid = c("rs17193061", "rs111473313", "rs115780507", "rs117392112"
    ), rv = c(126L, 135L, 137L, 132L)), class = "data.frame", row.names = c(NA, 
    4L)), NULL, NULL, NULL, list(date = "2008-05-27T05:41Z", 
        rsid = "rs61587769", rv = 130L), NULL, NULL, NULL, NULL, 
        structure(list(date = c("2003-08-27T10:59Z", "2003-12-4T15:58Z", 
        "2011-09-17T04:07Z", "2011-09-17T04:07Z", "2012-05-4T12:46Z"
        ), rsid = c("rs6908591", "rs9268538", "rs112333217", 
        "rs112804494", "rs116473074"), rv = c(117L, 119L, 135L, 
        135L, 137L)), class = "data.frame", row.names = c(NA, 
        5L)), structure(list(date = c("2003-08-27T10:59Z", "2003-12-4T15:58Z", 
        "2011-09-17T04:07Z", "2011-09-17T04:07Z", "2012-05-4T12:46Z"
        ), rsid = c("rs6908591", "rs9268538", "rs112333217", 
        "rs112804494", "rs116473074"), rv = c(117L, 119L, 135L, 
        135L, 137L)), class = "data.frame", row.names = c(NA, 
        5L)), structure(list(date = c("2008-05-25T14:17Z", "2014-08-21T16:14Z"
        ), rsid = c("rs59526942", "rs386602624"), rv = c(130L, 
        142L)), class = "data.frame", row.names = 1:2), NULL, 
        NULL, NULL, list(date = "2012-05-4T12:46Z", rsid = "rs113166067", 
            rv = 137L), NULL, NULL, NULL, NULL, NULL, list(date = "2008-05-25T15:53Z", 
            rsid = "rs59644354", rv = 130L), list(date = "2008-05-25T15:53Z", 
            rsid = "rs59644354", rv = 130L), NULL, NULL, structure(list(
            date = c("2004-10-8T03:35Z", "2008-05-24T09:53Z"), 
            rsid = c("rs17084453", "rs58049386"), rv = c(123L, 
            130L)), class = "data.frame", row.names = 1:2), structure(list(
            date = c("2004-10-8T03:35Z", "2008-05-24T09:53Z"), 
            rsid = c("rs17084453", "rs58049386"), rv = c(123L, 
            130L)), class = "data.frame", row.names = 1:2), list(
            date = "2008-05-24T09:38Z", rsid = "rs58045962", 
            rv = 130L), NULL, NULL, NULL, NULL, structure(list(
            date = c("2002-07-3T13:17Z", "2006-03-11T00:31Z", 
            "2006-03-11T00:31Z", "2008-05-25T15:51Z"), rsid = c("rs3192457", 
            "rs17855947", "rs17856982", "rs59642561"), rv = c(106L, 
            126L, 126L, 130L)), class = "data.frame", row.names = c(NA, 
        4L)), structure(list(date = c("2002-07-3T13:17Z", "2006-03-11T00:31Z", 
        "2006-03-11T00:31Z", "2008-05-25T15:51Z"), rsid = c("rs3192457", 
        "rs17855947", "rs17856982", "rs59642561"), rv = c(106L, 
        126L, 126L, 130L)), class = "data.frame", row.names = c(NA, 
        4L)), NULL, NULL, NULL, list(date = "2008-05-26T17:03Z", 
            rsid = "rs61065456", rv = 130L)), dbsnp.citations = list(
        NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 
        NULL, NULL, NULL, c(20383146L, 21931561L, 21980298L, 
        23028791L, 25238615L), 24281368L, 24281368L, NULL, NULL, 
        NULL, NULL, 17701893L, 11102985L, 24942156L, NULL, NULL, 
        c(1371116L, 2514102L, 8370580L, 17919309L), c(1371116L, 
        2514102L, 8370580L, 17919309L), NULL, NULL, NULL, NULL, 
        NULL, NULL, NULL, c(21681796L, 25586835L, 27583843L), 
        c(21681796L, 25586835L, 27583843L), NULL, NULL, 20022194L, 
        29590102L), dbsnp.hg19.end = c(43460113L, 31334852L, 
    88803982L, 88803982L, 88803982L, 42012159L, 228353840L, 228353840L, 
    10020564L, 160070943L, 32383441L, 32383441L, 176817636L, 
    10426672L, 10426672L, 126605316L, 44026548L, 228353833L, 
    228353833L, 113499003L, 41655209L, 8386415L, 85415734L, 85415734L, 
    73610978L, 73610978L, 55408875L, 55408875L, 39048163L, 26206299L, 
    26206299L, 80816411L, 80816411L, 156202173L, 156202173L, 
    75639578L, 75639578L, 236917290L, 163667L), dbsnp.hg19.start = c(43460113L, 
    31334852L, 88803982L, 88803982L, 88803982L, 42012159L, 228353840L, 
    228353840L, 10020564L, 160070943L, 32383441L, 32383441L, 
    176817636L, 10426672L, 10426672L, 126605316L, 44026548L, 
    228353833L, 228353833L, 113499002L, 41655209L, 8386415L, 
    85415734L, 85415734L, 73610978L, 73610978L, 55408875L, 55408875L, 
    39048163L, 26206299L, 26206299L, 80816411L, 80816411L, 156202173L, 
    156202173L, 75639578L, 75639578L, 236917290L, 163667L)), 
    elementType = "ANY", elementMetadata = NULL, metadata = list())
